resource_lists/astroneer/resources.yaml

```diff
@@ -2,531 +2,2257 @@
 authors:
   Asher: "https://aglick.com"
   Kevin: ""
+  Alban: "https://github.com/mopolita"
 
 index_page_display_name: "Astroneer"
 # row_group_count: 0
 
 recipe_types:
   Backpack Printer: "Use a Backpack Printer to turn {IN_ITEMS} into {OUT_ITEM}"
-  Smelter: "Use a Smelter to turn {IN_ITEMS} into {OUT_ITEM}"
-  Large Platform: "Use a Large Platform to turn {IN_ITEMS} into {OUT_ITEM}"
-  Printer: "Use a Printer to turn {IN_ITEMS} into {OUT_ITEM}"
-  Vehicle Bay: "Use a Vehicle Bay to turn {IN_ITEMS} into {OUT_ITEM}"
-  Printer and Vehicle Bay: "Use a Printer and Vehicle Bay to turn {IN_ITEMS} into {OUT_ITEM}"
+  Small Printer: "Use a Small Printer to turn {IN_ITEMS} into {OUT_ITEM}"
+  Medium Printer: "Use a Medium Printer to turn {IN_ITEMS} into {OUT_ITEM}"
+  Large Printer: "Use a Large Printer to turn {IN_ITEMS} into {OUT_ITEM}"
+  Smelting Furnace: "Use a Smelting Furnace to smelt {IN_ITEMS} into {OUT_ITEM}"
+  Chemistry Lab: "Use a Chemistry Lab to turn {IN_ITEMS} into {OUT_ITEM}"
+  Soil Centrifuge: "Use a Soil Centrifuge to make {OUT_ITEM} from {IN_ITEM}"
+  Trade Platform: "Trade {IN_ITEM} for {OUT_ITEM}"
 
 resources:
-  Coal:
+  Soil:
     recipes:
     - output: 1
       recipe_type: Raw Resource
       requirements:
-        Coal: 0
+        Soil: 0
 
   Compound:
     recipes:
     - output: 1
-      recipe_type: "Raw Resource"
+      recipe_type: Raw Resource
```

Comment:
For raw resource recipes a shorthand of just ` - recipe_type: Raw Resource` can be used in the recipe array (helps keep the size of the yaml file down).

Here's a recent example:
https://github.com/AsherGlick/ResourceCalculator/pull/98/files#diff-ab0edbea4a435a0b568d6309d9eed5bda445cbee5762645edd16cba6856aa365R21

Reply:
Just to clarify: This is not required, but it can make writing the yaml easier.